src/pages/all-on-one-v2/all-on-one-v2.ts

```diff
@@ -113,35 +122,77 @@ export class AllOnOneV2Page {
     return this.selectedManoeuvre === key;
   }
 
-  controlledStopPress() {
-    if (this.isControlledStopDone) return;
-    const { serious, dangerous } = this.controlledStopEl;
-    if (this.controlledStopEl.faultCounter > 0) {
+  // Show me button actions
+
+  setUpShowMeButton(tellMeFault: string) {
+    const noFault = undefined;
+    if (tellMeFault === noFault) {
       return;
     }
-    if (serious || dangerous) {
-      if (!this.isControlledStopDone) {
-        this.isControlledStopDone = !this.isControlledStopDone;
-        this.summaryMetaDataService.toggleControlledStopComplete();
-      }
+
+    const drivingFault = 'fault';
+    if (tellMeFault === drivingFault) {
+      this.showMeEl.addDrivingFault();
       return;
     }
+
+    const seriousFault = 'serious';
+    if (tellMeFault === seriousFault) {
+      this.showMeEl.addSeriousFault();
+      return;
+    }
+
+    const dangerousFault = 'dangerous';
+    if (tellMeFault === dangerousFault) {
+      this.showMeEl.addDangerousFault();
+      return;
+    }
+  }
+
+  showMePress() {
+    if (this.isShowMeDone) return;
+    if (this.showMeEl.faultCounter > 0) return;
+
+    const { serious, dangerous } = this.showMeEl;
+
+    if ((serious || dangerous) && this.isShowMeDone) return;
+
+    this.isShowMeDone = !this.isShowMeDone;
+    this.summaryMetaDataService.toggleShowMeComplete();
+  }
+
+  showMeTap() {
+    if (this.isShowMeDone && this.showMeEl.faultCounter > 0) return;
```

Comment:
Heard you discussing the difference between press and tap for this scenario yesterday, can explain why both are needed? What's the intended behaviour? I'm guessing you can do a quick tap to mark as done, but hold to mark a fault?

Reply:
Don't ask 😩

Reply:
Perhaps one to discuss in person...